docs/io.cozy.contacts.md

```diff
@@ -20,8 +20,7 @@ The `io.cozy.contacts` doctype is loosely based on the [vCard RFC](https://tools
 - `note?`: {string}
 - `email?`: {array} An array of email addresses objects with the following attributes:
   - `address`: {string} Email address
-  - `type?`: {string} Programmatic type of email (`"work"`, `"home"`, `"other"`)
-  - `label?`: {string} A user-provided localized type of email (example: `"Work"`)
+  - `type?`: {string} A user-provided localized type of phone number (example: `"Work"`)
```

Comment:
why are you removing `home` or `other`?

Reply:
My bad, it was rather a clumsy copy/paste